crates/db_views_actor/src/community_view.rs

```diff
@@ -29,39 +32,48 @@ impl CommunityView {
     community_id: CommunityId,
     my_person_id: Option<PersonId>,
   ) -> Result<Self, Error> {
-    // The left join below will return None in this case
-    let person_id_join = my_person_id.unwrap_or(PersonId(-1));
-
-    let (community, counts, follower, blocked) = community::table
-      .find(community_id)
-      .inner_join(community_aggregates::table)
-      .left_join(
-        community_follower::table.on(
-          community::id
-            .eq(community_follower::community_id)
-            .and(community_follower::person_id.eq(person_id_join)),
-        ),
-      )
-      .left_join(
-        community_block::table.on(
-          community::id
-            .eq(community_block::community_id)
-            .and(community_block::person_id.eq(person_id_join)),
-        ),
-      )
-      .select((
-        Community::safe_columns_tuple(),
-        community_aggregates::all_columns,
-        community_follower::all_columns.nullable(),
-        community_block::all_columns.nullable(),
-      ))
-      .first::<CommunityViewTuple>(conn)?;
-
-    Ok(CommunityView {
-      community,
-      subscribed: CommunityFollower::to_subscribed_type(&follower),
-      blocked: blocked.is_some(),
-      counts,
+    conn.build_transaction().read_write().run(|| {
+      // The left join below will return None in this case
+      let person_id_join = my_person_id.unwrap_or(PersonId(-1));
+
+      let (community, counts, follower, blocked) = community::table
+          .find(community_id)
+          .inner_join(community_aggregates::table)
+          .left_join(
+            community_follower::table.on(
+              community::id
+                  .eq(community_follower::community_id)
+                  .and(community_follower::person_id.eq(person_id_join)),
+            ),
+          )
+          .left_join(
+            community_block::table.on(
+              community::id
+                  .eq(community_block::community_id)
+                  .and(community_block::person_id.eq(person_id_join)),
+            ),
+          )
+          .select((
+            Community::safe_columns_tuple(),
+            community_aggregates::all_columns,
+            community_follower::all_columns.nullable(),
+            community_block::all_columns.nullable(),
+          ))
+          .first::<CommunityViewTuple>(conn)?;
+
+      let languages = community::table
```

Comment:
Same here, move these to GetCommunityResponse. You can tell this is bad because you have this for read, but not for the query builder.